src/utils/express.ts

```diff
@@ -9,10 +9,11 @@ import {
  * which takes an express.Request as input and returns an IResponse,
  * into an express controller.
  */
-export function toExpressHandler<T>(
-  handler: (req: express.Request) => Promise<IResponse<T>>
-): <P>(req: express.Request, res: express.Response, object?: P) => void {
-  return async (req, res, object) => {
+export function toExpressHandler<T, P>(
+  handler: (req: express.Request) => Promise<IResponse<T>>,
+  object?: P
+): (req: express.Request, res: express.Response) => void {
+  return async (req, res) => {
```

Comment:
don't use `async` here, Express doesn't support async handlers - use then/catch to handle the promise returned by `handler`

Reply:
fixed in c1c6437